.azure-pipelines/templates/variables.yml

```diff
@@ -1,5 +1,6 @@
 variables:
   ubuntu_pool: 'pool-ubuntu-2004'
+  ubuntu_multi_core_pool: 'pool-ubuntu-2004-multi-core'
```

Comment:
`{{ variables.ubuntu_multi_core_pool }}` is not used, included here for reference.

PS: We use `architectures.pool` only.
https://github.com/Azure/azure-cli/blob/b4efda65563d320dfda683051c2543332faf4f0d/azure-pipelines.yml#L23-L31